doc/dune-files.rst

```diff
@@ -584,30 +580,30 @@ to use the :ref:`include_subdirs` stanza.
   that must be installed, you must list them in this field, without the ``.h``
   extension.
 
-- ``(modes <modes>)`` is for modes which should be built by default. The
-  most common use for this feature is to disable native compilation
-  when writing libraries for the OCaml toplevel. It can also be used to define
-  libraries that should be compiled with Melange (see :ref:`melange_main`). The
-  following modes are available: ``byte``, ``native``, ``best``, and ``melange``.
-  ``best`` is ``native`` or ``byte`` when native compilation isn't available. 
-
-- ``(no_dynlink)`` disables dynamic linking of the library. This is for
-  advanced use only. By default, you shouldn't set this option.
-
-- ``(kind <kind>)`` sets the type of library. The default is ``normal``, but other
-  available choices are ``ppx_rewriter`` and ``ppx_deriver``. They must be set
-  when the library is intended to be used as a ppx rewriter or a ``[@@deriving
-  ...]`` plugin. The reason ``ppx_rewriter`` and ``ppx_deriver`` are split
-  is historical, and hopefully we won't need two options soon. Both ppx kinds
-  support an optional field: ``(cookies <cookies>)``, where ``<cookies>`` is a
-  list of pairs ``(<name> <value>)`` with ``<name>`` being the cookie name and
-  ``<value>`` a string that supports :ref:`variables` evaluated
-  by each preprocessor invocation (note: libraries that share
-  cookies with the same name should agree on their expanded value).
-
-- ``(ppx_runtime_libraries (<library-names>))`` is for when the library is a ``ppx
-  rewriter`` or a ``[@@deriving ...]`` plugin, and has runtime dependencies. You
-  need to specify these runtime dependencies here.
+- ``(modes <modes>)`` is for modes which should be built by default. The most
+  common use for this feature is to disable native compilation when writing
+  libraries for the OCaml toplevel. It can also be used to define libraries that
+  should be compiled with Melange (see :ref:`melange_main`). The following modes
+  are available: ``byte``, ``native``, ``best``, and ``melange``. ``best`` is
+  ``native`` or ``byte`` when native compilation isn't available. 
+
+- ``(no_dynlink)`` disables dynamic linking of the library. This is for advanced
+  use only. By default, you shouldn't set this option.
+
+- ``(kind <kind>)`` sets the type of library. The default is ``normal``, but
+  other available choices are ``ppx_rewriter`` and ``ppx_deriver``. They must be
+  set when the library is intended to be used as a ppx rewriter or a
```

Comment:
PPX rewriter. 
PPX capitalized unless wrapped in monospace for commands, etc.

Reply:
I believe ppx is an acronym that gets to stay lowercase.

Reply:
In our style guide is says it's to be PPX. I think I checked with the maintainers back when it was put in there. I've been capitalising it in docs and blog posts.

Reply:
Even before I got here in [this post](https://tarides.com/blog/2019-05-09-an-introduction-to-ocaml-ppx-ecosystem). It's also capitalised [here on OCaml.org](https://ocaml.org/docs/metaprogramming) and [here on OCaml Verse](http://ocamlverse.net/content/ppx.html), but there are lots of places it isn't, too. Let's confirm the official formatting, so we can make everything consistent as we go through the documentation and write new things.